pkg/util/containers/cri/util.go

```diff
@@ -104,6 +108,53 @@ func GetUtil() (*CRIUtil, error) {
 	return globalCRIUtil, nil
 }
 
+// sts begin
+
+// ContainerStateMap is used to map cri specific state to own internal state
+var ContainerStateMap = map[pb.ContainerState]string{
+	pb.ContainerState_CONTAINER_CREATED: containers.ContainerCreatedState,
+	pb.ContainerState_CONTAINER_RUNNING: containers.ContainerRunningState,
+	pb.ContainerState_CONTAINER_EXITED:  containers.ContainerExitedState,
+	pb.ContainerState_CONTAINER_UNKNOWN: containers.ContainerUnknownState,
+}
+
+func (c *CRIUtil) GetContainers() ([]*spec.Container, error) {
+	containerStats, err := c.ListContainerStats()
+	if err != nil {
+		return nil, err
+	}
+	uContainers := make([]*spec.Container, 0, len(containerStats))
+	for cid := range containerStats {
+		cstatus, err := c.GetContainerStatus(cid)
+		if err != nil {
+			log.Debugf("Could not get status of container '%s'", cid)
```

Comment:
`ContainerD` logs these as errors and I think they should be consistent. To be honest I think probably `warn` is the right level.